scipy/ndimage/src/_ni_label.pyx

```diff
@@ -271,7 +271,7 @@ cpdef _label(np.ndarray input,
 
     # we only process this many neighbors from the itstruct iterator before
     # reaching the center, where we stop
-    num_neighbors = (structure.size / 3) // 2
+    num_neighbors = (structure.size // 3) // 2
```

Comment:
I am not sure about this one - should it be more explicit about how the float value `structure.size / 3` is rounded to int? Should it be `int(structure.size / 3 / 2)` ?

Reply:
A few lines above there is a statement that claims to check that `structure.size` is divisible by 3.

Reply:
>I am asking what the intent of the original code was,

I'm not sure I see where the ambiguity is, consider
```python
for i in range(10000):
    assert int(i / 3 / 2) == (i / 3) // 2 == (i // 3) // 2 == i // 6 
```
When I run this it passes. I think these are all equivalent (at least for `i >= 0`), so maybe just go with the simplest version

```suggestion
    num_neighbors = structure.size // 6
```